pulsar-broker/src/test/java/org/apache/pulsar/schema/compatibility/SchemaTypeCompatibilityCheckTest.java

```diff
@@ -185,6 +204,7 @@ public void structTypeProducerConsumerUndefinedCompatible() throws Exception {
     @Test
     public void structTypeConsumerProducerUndefinedCompatible() throws Exception {
         admin.namespaces().setSchemaCompatibilityStrategy(namespaceName, SchemaCompatibilityStrategy.UNDEFINED);
+        setTopicLevelPoliciesEnabled(false);
```

Comment:
It seems that every unit test needs to be set up like this, why not move it to `setup()`?